src/plots/cartesian/graph_interact.js

```diff
@@ -927,6 +927,13 @@ function createHoverText(hoverData, opts) {
             if(name.length>15) name = name.substr(0,12)+'...';
         }
 
+        // used by other modules (initially just ternary) that
+        // manage their own hoverinfo independent of cleanPoint
+        // the rest of this will still apply, so such modules
+        // can still put things in (x|y|z)Label, text, and name
+        // and hoverinfo will still determine their visibility
+        if(d.extraText!==undefined) text += d.extraText;
```

Comment:
the next step in *The Great Graph Interact Breakup* would be to make an `components/hoverlabels/` modules, general enough to encompass all of plot types.